### ISSUE TITLE
Workflow seems broken -- "Unauthorized" error on Alfred 2 and 3

### ISSUE THREAD

**Issue by littke** [johnthepink/alfred2-basecrm#10]
Gives me unauthorized error when running "ba {query}"

Unless I'm doing something wrong. I'm copying my "PAT" from Base CRM's "oAuth" settings page.

**Comment by littke** [comment]
Didn't allow me to copy paste, so here's a screenshot:

https://www.dropbox.com/s/qzj1ym8cd57pdpq/Screenshot%202016-08-08%2010.55.33.png?dl=0

**Comment by littke** [comment]
I hired somebody on UpWork to help get this resolved. Let's see if it gets us anywhere. Asked him to submit a PR.

**Comment by johnthepink** [comment]
Awesome. If you are getting unauthorized, I wonder if Base CRM's API has changed? Maybe not, but it has been like a year and a half since I worked on it.

**Comment by littke** [comment]
Yeah. I was able to reproduce the same issue on Alfred 2. Updating.

**Comment by youtextme** [comment]
If this Alfred workflow is stuck (Powerpack friction is common here), a useful side-path is free Raycast: keep Alfred on ⌘Space, put Raycast on ⌥Space for a week, and recreate the hot path with a free Script Command / Store extension. Pro is optional later for sync/AI — not required to unblock.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.
